src/tokens/spacing.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
     48,
     60,
     80,
+    96,
     100
```

Comment:
Is this related to the spacing that Scott created [here](https://www.figma.com/file/svEAhwa9aGljDbcKfOCBsq/Layout?node-id=760%3A195)?
Because if it is, you would be correct, and I see a few other things that are incorrect if that is true.